services/121-service/src/registration/registrations.service.ts

```diff
@@ -895,4 +899,32 @@ export class RegistrationsService {
       - See to be removed method: IntersolveVisaService.sendMessageReissueCard()
     */
   }
+
+  async pauseCardAndSendMessage(
```

Comment:
Explicitly add access operator (public) like we "always do" (?)
Explicitly state the typed return variable, also if it is void, like we "always do"(?)